tools/cldr-code/src/license/THIRD-PARTY.properties

```diff
@@ -1,6 +1,16 @@
 # Generated by org.codehaus.mojo.license.AddThirdPartyMojo
+#-------------------------------------------------------------------------------
+# Already used licenses in project :
+# - Apache 2.0
+# - Apache License, Version 2.0
+# - The Apache Software License, Version 2.0
+# - The MIT License
+# - The SAX License
+# - The W3C License
+#-------------------------------------------------------------------------------
 # Please fill the missing licenses for dependencies :
-# this may need to be updated before building final outputs
-# (for now)
-com.ibm.icu--icu4j-for-cldr--68.1-release-68-1=Unicode-DFS-2016
-com.ibm.icu--utilities-for-cldr--68.1-release-68-1=Unicode-DFS-2016
+#
+#
+#Wed Feb 17 10:22:40 PST 2021
+com.ibm.icu--icu4j-for-cldr--69.1-SNAPSHOT-cldr-2021-02-17=
+com.ibm.icu--utilities-for-cldr--69.1-SNAPSHOT-cldr-2021-02-17=
```

Comment:
Do we need to add a reference to ICU or Unicode license somewhere to remove these lines?

Reply:
I think there is probably a way to put the correct license metadata into those files so it can be picked up.  I started a PR to do that on the ICU side.

The license is the same as CLDRs so should be covered.